packages/message-system/src/config/config.v1.json

```diff
@@ -346,23 +346,39 @@
                             "variant": "*",
                             "firmwareRevision": "*",
                             "vendor": "*"
+                        },
+                        {
+                            "model": "1",
+                            "firmware": "*",
+                            "bootloader": "*",
+                            "variant": "*",
+                            "firmwareRevision": "661ae37506f6e6ef94067d4b6de26c80bfb4c4b5",
+                            "vendor": "*"
+                        },
+                        {
+                            "model": "T",
+                            "firmware": "*",
+                            "bootloader": "*",
+                            "variant": "*",
+                            "firmwareRevision": "661ae37506f6e6ef94067d4b6de26c80bfb4c4b5",
+                            "vendor": "*"
                         }
                     ]
                 }
             ],
             "message": {
                 "id": "6f793f82-5e13-492f-803c-100c8dcc3cb6",
```

Comment:
one nitpick as usual - would be nice to change the message id so it will reappear to those who already dismissed it

Reply:
Good point 👍